pkg/template/templateprocessing/template.go

```diff
@@ -14,6 +14,7 @@ import (
 	appsv1 "github.com/openshift/api/apps/v1"
 	templatev1 "github.com/openshift/api/template/v1"
 	. "github.com/openshift/library-go/pkg/template/generator"
+
 	"github.com/openshift/origin/pkg/api/legacygroupification"
```

Comment:
@deads2k ^ this thing exists to deal with legacy templates that has non-groupified resources... we discussed with @soltysh whether it makes sense to move it to `openshift/api`.